emu/src/memory/internal_memory.rs

```diff
@@ -209,7 +215,7 @@ impl InternalMemory {
 }
 
 impl IoDevice for InternalMemory {
-    type Address = u32;
+    type Address = usize;
     type Value = u8;
 
     fn read_at(&self, address: Self::Address) -> Self::Value {
```

Comment:
We should add the case when we read from the ROM mapped area, right? (The one starting at `0x08...`)